latest/Dockerfile.rhel7

```diff
@@ -30,7 +30,7 @@ ENV MONGODB_VERSION=3.4 \
 EXPOSE 27017
 
 ENTRYPOINT ["container-entrypoint"]
-CMD ["run-mongod"]
+CMD ["run-standalone"]
```

Comment:
I remember that we had to keep `run-mongod-replset` as symlink to `run-mongod-replication` after renaming. Because users have this command already in the templates, ...
So not sure if we don't have to do the same for `run-monod` if we decide to rename it.
@jbornemann Any background story/rasoning behind your naming scheme? (`run-standalone`, `run-configsrv`, `run-replset`,...)

We chose `run-mongod`, `run-mongod-replication` and I can imagine `run-mongod-configsrv`, `run-mongos`,...

mariadb-container, mysql-container and postgresql-container uses same pattern `run-DAEMON[-ROLE]`. So `run-mysqld`, `run-mysqld-master` and `run-mysqld-slave`.

Reply:
Reasoning was to keep intent clear - almost all of the modes are running mongod, but *how* are they running mongod? Keeping consistent with the mariadb scheme is probably a good idea.